src/Microsoft.TestPlatform.CoreUtilities/Helpers/DotnetHostHelper.cs

```diff
@@ -118,8 +118,14 @@ private bool TryGetExecutablePath(string executableBaseName, out string executab
         return false;
     }
 
-    public bool TryGetDotnetPathByArchitecture(PlatformArchitecture targetArchitecture, [NotNullWhen(true)] out string? muxerPath)
+    public bool TryGetDotnetPathByArchitecture(
+        PlatformArchitecture targetArchitecture,
+        DotnetMuxerResolutionStrategy dotnetMuxerResolutionStrategy,
+        [NotNullWhen(true)] out string? muxerPath)
     {
+        muxerPath = null;
+        EqtTrace.Verbose($"DotnetHostHelper.TryGetDotnetPathByArchitecture: Using dotnet muxer resolution strategy: {dotnetMuxerResolutionStrategy}");
```

Comment:
can you verify that logs is clear here? without the strategy all other logs are complex to reason